docker/iota-data-ingestion/config/config.yaml

```diff
@@ -0,0 +1,60 @@
+# IndexerExecutor config
```

Comment:
why do you create this config file here in the `docker` folder? is it necessary to build the container?

Reply:
[fixed](https://github.com/iotaledger/iota/pull/5026/commits/1974930a2fc592d362a861f5c5c6ef8f50ea1587)